frontend/src/scenes/onboarding/productAnalyticsSteps/DashboardTemplateVariables.tsx

```diff
@@ -0,0 +1,186 @@
+import { IconCheckCircle, IconInfo, IconTrash } from '@posthog/icons'
+import { LemonBanner, LemonButton, LemonCollapse, LemonInput, LemonLabel } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { useEffect, useState } from 'react'
+import { dashboardTemplateVariablesLogic } from 'scenes/dashboard/dashboardTemplateVariablesLogic'
+import { newDashboardLogic } from 'scenes/dashboard/newDashboardLogic'
+
+import { DashboardTemplateVariableType } from '~/types'
+
+function VariableSelector({
+    variable,
+    hasSelectedSite,
+}: {
+    variable: DashboardTemplateVariableType
+    hasSelectedSite: boolean
+}): JSX.Element {
+    const { activeDashboardTemplate } = useValues(newDashboardLogic)
+    const theDashboardTemplateVariablesLogic = dashboardTemplateVariablesLogic({
+        variables: activeDashboardTemplate?.variables || [],
+    })
+    const { setVariable, resetVariable, goToNextUntouchedActiveVariableIndex, incrementActiveVariableIndex } =
+        useActions(theDashboardTemplateVariablesLogic)
+    const { allVariablesAreTouched, variables, activeVariableIndex } = useValues(theDashboardTemplateVariablesLogic)
+    const [customEventName, setCustomEventName] = useState<string | null>(null)
+    const [showCustomEventField, setShowCustomEventField] = useState(false)
+
+    const FALLBACK_EVENT = {
+        id: '$other_event',
+        math: 'dau',
+        type: 'events',
+    }
+
+    return (
+        <div className="pl-7">
+            <div className="mb-2">
+                <p>
+                    <IconInfo /> {variable.description}
+                </p>
+            </div>
+            {variable.touched && !customEventName && (
+                <div className="flex justify-between items-center bg-bg-3000-light p-2 pl-3 rounded mb-4">
+                    <div>
+                        <IconCheckCircle className="text-success font-bold" />{' '}
+                        <span className="text-success font-bold">Selected</span>
+                        <p className="italic text-muted mb-0">.md-invite-button</p>
```

Comment:
I don't understand the UI here, why is this hard coded?

Reply:
Just because I haven't actually hooked it up to the toolbar and iframe yet! That's coming in the next PR.

Reply:
Ah yeah you mentioned that, my bad!